spec/unit/block_score/actions/update_spec.rb

```diff
@@ -2,15 +2,41 @@
 
 RSpec.describe BlockScore::Actions::Update, vcr: true do
   describe '#save!' do
-    context 'when updating an existing candidate' do
-      subject(:candidate) { create(:candidate, name_first: 'John') }
+    subject(:candidate) { build(:candidate, name_first: 'John') }
+    context 'when updating an new candidate' do
       before do
-        candidate.name_first = 'Jane'
-        candidate.save!
+        expect(candidate.persisted?).to be false
+        expect(candidate.save!).to be true
+
+        expect(candidate.persisted?).to be true
+      end
+
+      it 'has the same attributes' do
+        expect(BlockScore::Candidate.retrieve(candidate.id)
+                 .attributes.reject { |key, _| key.equal?(:updated_at) })
+          .to eql(candidate.attributes.reject do |key, _|
+            key.equal?(:updated_at)
+          end)
       end
+    end
 
-      it { is_expected.to be_persisted }
-      its(:name_first) { is_expected.to eql 'Jane' }
+    context 'when changing first update candidate', vcr: { record: :all } do
+      it 'has the same attributes' do
+        expect(candidate.persisted?).to be false
+        expect(candidate.save!).to be true
```

Comment:
> In this particular case I cannot use the idiom, because I need to test the return value to satisfy killing a mutation.

As there is the `.to(true)` block the return value is verified, and changing the value to anything but `true` should be killed.

If the mutation is still alive using `expect { }.to change` might use a to permissive internal matching on expected values, that would be really unfortunate.